sos/policies/package_managers/__init__.py

```diff
@@ -112,13 +118,12 @@ def pkg_by_name(self, name):
         :returns: The first package that matches `name`
         :rtype: ``str``
         """
-        pkgmatches = self.all_pkgs_by_name(name)
-        if (len(pkgmatches) != 0):
-            return self.all_pkgs_by_name(name)[-1]
-        else:
+        try:
+            return self.packages[name]
+        except Exception:
             return None
 
-    def get_pkg_list(self):
+    def _get_pkg_list(self):
```

Comment:
Technically the method does not *get* the pkg list, but it "just" generates it. Worth changing its name? (and definitely change its description - that (both together) is confusing).

Reply:
Ack, fair point.